tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -560,8 +633,19 @@ def _wait_for_loggers(self, level, request_time, namespace, workers=None):
         )
 
     @cluster(num_nodes=6)
-    @matrix(security_protocol=[SecurityConfig.PLAINTEXT, SecurityConfig.SASL_SSL], exactly_once_source=[True, False], connect_protocol=['sessioned', 'compatible', 'eager'], metadata_quorum=quorum.all_non_upgrade)
-    def test_file_source_and_sink(self, security_protocol, exactly_once_source, connect_protocol, metadata_quorum):
+    @matrix(
+        security_protocol=[SecurityConfig.PLAINTEXT, SecurityConfig.SASL_SSL],
+        exactly_once_source=[True, False], connect_protocol=['sessioned', 'compatible', 'eager'],
```

Comment:
nit: Should `connect_protocol` be on a new line?

Reply:
oh yes thanks for the catch!